### ISSUE TITLE
Error on install- LINK : fatal error LNK1181: cannot open input file 'm.lib'

### ISSUE THREAD

**Issue by sajandeep** [iamDecode/sklearn-pmml-model#8]
When trying to pip install sklearn-pmml-model  I get fatal error LNK1181. 
See full details below.

```
C:\Users\sajandeep.sidhu>pip install sklearn-pmml-model
Collecting sklearn-pmml-model
  Using cached https://files.pythonhosted.org/packages/93/2c/8cecb62ad09ec87efaf9d59f92bbf6d745b00599f400ad5a8ac464f60f6d/sklearn-pmml-model-0.0.4.tar.gz
Requirement already satisfied: numpy in c:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\lib\site-packages (from sklearn-pmml-model) (1.14.0)
Requirement already satisfied: pandas in c:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\lib\site-packages (from sklearn-pmml-model) (0.22.0)
Requirement already satisfied: scipy in c:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\lib\site-packages (from sklearn-pmml-model) (1.0.0)
Requirement already satisfied: scikit-learn in c:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\lib\site-packages (from sklearn-pmml-model) (0.19.1)
Requirement already satisfied: cached-property in c:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\lib\site-packages (from sklearn-pmml-model) (1.5.1)
Requirement already satisfied: python-dateutil>=2 in c:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\lib\site-packages (from pandas->sklearn-pmml-model) (2.6.1)
Requirement already satisfied: pytz>=2011k in c:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\lib\site-packages (from pandas->sklearn-pmml-model) (2017.3)
Requirement already satisfied: six>=1.5 in c:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\lib\site-packages (from python-dateutil>=2->pandas->sklearn-pmml-model) (1.11.0)
Building wheels for collected packages: sklearn-pmml-model
  Building wheel for sklearn-pmml-model (setup.py) ... error
  Complete output from command c:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\python.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\SAJAND~1.SID\\AppData\\Local\\Temp\\pip-install-rww4zo40\\sklearn-pmml-model\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d C:\Users\SAJAND~1.SID\AppData\Local\Temp\pip-wheel-5udid9h5 --python-tag cp36:
  build configuration selected: optimized
  c:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\lib\distutils\dist.py:261: UserWarning: Unknown distribution option: 'long_description_content_type'
    warnings.warn(msg)
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build\lib.win-amd64-3.6
  creating build\lib.win-amd64-3.6\sklearn_pmml_model
  copying sklearn_pmml_model\base.py -> build\lib.win-amd64-3.6\sklearn_pmml_model
  copying sklearn_pmml_model\datatypes.py -> build\lib.win-amd64-3.6\sklearn_pmml_model
  copying sklearn_pmml_model\__init__.py -> build\lib.win-amd64-3.6\sklearn_pmml_model
  creating build\lib.win-amd64-3.6\tests
  copying tests\test_base.py -> build\lib.win-amd64-3.6\tests
  copying tests\test_datatypes.py -> build\lib.win-amd64-3.6\tests
  copying tests\__init__.py -> build\lib.win-amd64-3.6\tests
  creating build\lib.win-amd64-3.6\sklearn_pmml_model\ensemble
  copying sklearn_pmml_model\ensemble\forest.py -> build\lib.win-amd64-3.6\sklearn_pmml_model\ensemble
  copying sklearn_pmml_model\ensemble\__init__.py -> build\lib.win-amd64-3.6\sklearn_pmml_model\ensemble
  creating build\lib.win-amd64-3.6\sklearn_pmml_model\tree
  copying sklearn_pmml_model\tree\tree.py -> build\lib.win-amd64-3.6\sklearn_pmml_model\tree
  copying sklearn_pmml_model\tree\__init__.py -> build\lib.win-amd64-3.6\sklearn_pmml_model\tree
  copying sklearn_pmml_model\tree\quad_tree.pxd -> build\lib.win-amd64-3.6\sklearn_pmml_model\tree
  copying sklearn_pmml_model\tree\_criterion.pxd -> build\lib.win-amd64-3.6\sklearn_pmml_model\tree
  copying sklearn_pmml_model\tree\_splitter.pxd -> build\lib.win-amd64-3.6\sklearn_pmml_model\tree
  copying sklearn_pmml_model\tree\_tree.pxd -> build\lib.win-amd64-3.6\sklearn_pmml_model\tree
  copying sklearn_pmml_model\tree\_utils.pxd -> build\lib.win-amd64-3.6\sklearn_pmml_model\tree
  copying sklearn_pmml_model\tree\quad_tree.pyx -> build\lib.win-amd64-3.6\sklearn_pmml_model\tree
  copying sklearn_pmml_model\tree\_criterion.pyx -> build\lib.win-amd64-3.6\sklearn_pmml_model\tree
  copying sklearn_pmml_model\tree\_splitter.pyx -> build\lib.win-amd64-3.6\sklearn_pmml_model\tree
  copying sklearn_pmml_model\tree\_tree.pyx -> build\lib.win-amd64-3.6\sklearn_pmml_model\tree
  copying sklearn_pmml_model\tree\_utils.pyx -> build\lib.win-amd64-3.6\sklearn_pmml_model\tree
  running build_ext
  building 'sklearn_pmml_model.tree._tree' extension
  creating build\temp.win-amd64-3.6
  creating build\temp.win-amd64-3.6\Release
  creating build\temp.win-amd64-3.6\Release\sklearn_pmml_model
  creating build\temp.win-amd64-3.6\Release\sklearn_pmml_model\tree
  C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Tools\MSVC\14.16.27023\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -I. -Ic:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\lib\site-packages\numpy\core\include -Ic:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\include -Ic:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Tools\MSVC\14.16.27023\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\cppwinrt" /Tcsklearn_pmml_model\tree\_tree.c /Fobuild\temp.win-amd64-3.6\Release\sklearn_pmml_model\tree\_tree.obj -march=native -O2 -msse -msse2 -mfma -mfpmath=sse
  cl : Command line warning D9002 : ignoring unknown option '-march=native'
  cl : Command line warning D9002 : ignoring unknown option '-msse'
  cl : Command line warning D9002 : ignoring unknown option '-msse2'
  cl : Command line warning D9002 : ignoring unknown option '-mfma'
  cl : Command line warning D9002 : ignoring unknown option '-mfpmath=sse'
  _tree.c
  c:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\lib\site-packages\numpy\core\include\numpy\npy_1_7_deprecated_api.h(12) : Warning Msg: Using deprecated NumPy API, disable it by #defining NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
  sklearn_pmml_model\tree\_tree.c(5266): warning C4244: 'function': conversion from '__pyx_t_18sklearn_pmml_model_4tree_5_tree_SIZE_t' to 'long', possible loss of data
  sklearn_pmml_model\tree\_tree.c(7433): warning C4244: '=': conversion from '__pyx_t_18sklearn_pmml_model_4tree_5_tree_SIZE_t' to '__pyx_t_18sklearn_pmml_model_4tree_6_utils_DOUBLE_t', possible loss of data
  sklearn_pmml_model\tree\_tree.c(11751): warning C4244: '=': conversion from '__pyx_t_18sklearn_pmml_model_4tree_5_tree_SIZE_t' to '__pyx_t_18sklearn_pmml_model_4tree_6_utils_DOUBLE_t', possible loss of data
  sklearn_pmml_model\tree\_tree.c(14037): warning C4244: 'function': conversion from '__pyx_t_18sklearn_pmml_model_4tree_5_tree_SIZE_t' to 'long', possible loss of data
  sklearn_pmml_model\tree\_tree.c(14101): warning C4244: 'function': conversion from '__pyx_t_18sklearn_pmml_model_4tree_5_tree_SIZE_t' to 'long', possible loss of data
  sklearn_pmml_model\tree\_tree.c(14309): warning C4244: '=': conversion from '__pyx_t_18sklearn_pmml_model_4tree_5_tree_SIZE_t' to 'long', possible loss of data
  sklearn_pmml_model\tree\_tree.c(14388): warning C4244: '=': conversion from '__pyx_t_18sklearn_pmml_model_4tree_5_tree_SIZE_t' to 'long', possible loss of data
  sklearn_pmml_model\tree\_tree.c(14965): warning C4244: 'function': conversion from '__pyx_t_18sklearn_pmml_model_4tree_5_tree_SIZE_t' to 'long', possible loss of data
  sklearn_pmml_model\tree\_tree.c(15029): warning C4244: 'function': conversion from '__pyx_t_18sklearn_pmml_model_4tree_5_tree_SIZE_t' to 'long', possible loss of data
  sklearn_pmml_model\tree\_tree.c(15331): warning C4244: '=': conversion from '__pyx_t_18sklearn_pmml_model_4tree_5_tree_SIZE_t' to 'long', possible loss of data
  sklearn_pmml_model\tree\_tree.c(15451): warning C4244: '=': conversion from '__pyx_t_18sklearn_pmml_model_4tree_5_tree_SIZE_t' to 'long', possible loss of data
  sklearn_pmml_model\tree\_tree.c(16466): warning C4133: 'function': incompatible types - from 'PyObject *' to 'PyTypeObject *'
  sklearn_pmml_model\tree\_tree.c(24679): warning C4244: 'initializing': conversion from 'double' to 'float', possible loss of data
  sklearn_pmml_model\tree\_tree.c(24685): warning C4244: 'initializing': conversion from 'double' to 'float', possible loss of data
  C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Tools\MSVC\14.16.27023\bin\HostX86\x64\link.exe /nologo /INCREMENTAL:NO /LTCG /DLL /MANIFEST:EMBED,ID=2 /MANIFESTUAC:NO /LIBPATH:c:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\libs /LIBPATH:c:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\PCbuild\amd64 "/LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Tools\MSVC\14.16.27023\lib\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.17763.0\ucrt\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.17763.0\um\x64" m.lib /EXPORT:PyInit__tree build\temp.win-amd64-3.6\Release\sklearn_pmml_model\tree\_tree.obj /OUT:build\lib.win-amd64-3.6\sklearn_pmml_model\tree\_tree.cp36-win_amd64.pyd /IMPLIB:build\temp.win-amd64-3.6\Release\sklearn_pmml_model\tree\_tree.cp36-win_amd64.lib
  LINK : fatal error LNK1181: cannot open input file 'm.lib'
  error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\BuildTools\\VC\\Tools\\MSVC\\14.16.27023\\bin\\HostX86\\x64\\link.exe' failed with exit status 1181

  ----------------------------------------
  Failed building wheel for sklearn-pmml-model
  Running setup.py clean for sklearn-pmml-model
Failed to build sklearn-pmml-model
Installing collected packages: sklearn-pmml-model
  Running setup.py install for sklearn-pmml-model ... error
    Complete output from command c:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\python.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\SAJAND~1.SID\\AppData\\Local\\Temp\\pip-install-rww4zo40\\sklearn-pmml-model\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record C:\Users\SAJAND~1.SID\AppData\Local\Temp\pip-record-ez85ewko\install-record.txt --single-version-externally-managed --compile:
    build configuration selected: optimized
    c:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\lib\distutils\dist.py:261: UserWarning: Unknown distribution option: 'long_description_content_type'
      warnings.warn(msg)
    running install
    running build
    running build_py
    creating build
    creating build\lib.win-amd64-3.6
    creating build\lib.win-amd64-3.6\sklearn_pmml_model
    copying sklearn_pmml_model\base.py -> build\lib.win-amd64-3.6\sklearn_pmml_model
    copying sklearn_pmml_model\datatypes.py -> build\lib.win-amd64-3.6\sklearn_pmml_model
    copying sklearn_pmml_model\__init__.py -> build\lib.win-amd64-3.6\sklearn_pmml_model
    creating build\lib.win-amd64-3.6\tests
    copying tests\test_base.py -> build\lib.win-amd64-3.6\tests
    copying tests\test_datatypes.py -> build\lib.win-amd64-3.6\tests
    copying tests\__init__.py -> build\lib.win-amd64-3.6\tests
    creating build\lib.win-amd64-3.6\sklearn_pmml_model\ensemble
    copying sklearn_pmml_model\ensemble\forest.py -> build\lib.win-amd64-3.6\sklearn_pmml_model\ensemble
    copying sklearn_pmml_model\ensemble\__init__.py -> build\lib.win-amd64-3.6\sklearn_pmml_model\ensemble
    creating build\lib.win-amd64-3.6\sklearn_pmml_model\tree
    copying sklearn_pmml_model\tree\tree.py -> build\lib.win-amd64-3.6\sklearn_pmml_model\tree
    copying sklearn_pmml_model\tree\__init__.py -> build\lib.win-amd64-3.6\sklearn_pmml_model\tree
    copying sklearn_pmml_model\tree\quad_tree.pxd -> build\lib.win-amd64-3.6\sklearn_pmml_model\tree
    copying sklearn_pmml_model\tree\_criterion.pxd -> build\lib.win-amd64-3.6\sklearn_pmml_model\tree
    copying sklearn_pmml_model\tree\_splitter.pxd -> build\lib.win-amd64-3.6\sklearn_pmml_model\tree
    copying sklearn_pmml_model\tree\_tree.pxd -> build\lib.win-amd64-3.6\sklearn_pmml_model\tree
    copying sklearn_pmml_model\tree\_utils.pxd -> build\lib.win-amd64-3.6\sklearn_pmml_model\tree
    copying sklearn_pmml_model\tree\quad_tree.pyx -> build\lib.win-amd64-3.6\sklearn_pmml_model\tree
    copying sklearn_pmml_model\tree\_criterion.pyx -> build\lib.win-amd64-3.6\sklearn_pmml_model\tree
    copying sklearn_pmml_model\tree\_splitter.pyx -> build\lib.win-amd64-3.6\sklearn_pmml_model\tree
    copying sklearn_pmml_model\tree\_tree.pyx -> build\lib.win-amd64-3.6\sklearn_pmml_model\tree
    copying sklearn_pmml_model\tree\_utils.pyx -> build\lib.win-amd64-3.6\sklearn_pmml_model\tree
    running build_ext
    building 'sklearn_pmml_model.tree._tree' extension
    creating build\temp.win-amd64-3.6
    creating build\temp.win-amd64-3.6\Release
    creating build\temp.win-amd64-3.6\Release\sklearn_pmml_model
    creating build\temp.win-amd64-3.6\Release\sklearn_pmml_model\tree
    C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Tools\MSVC\14.16.27023\bin\HostX86\x64\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -I. -Ic:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\lib\site-packages\numpy\core\include -Ic:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\include -Ic:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Tools\MSVC\14.16.27023\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\cppwinrt" /Tcsklearn_pmml_model\tree\_tree.c /Fobuild\temp.win-amd64-3.6\Release\sklearn_pmml_model\tree\_tree.obj -march=native -O2 -msse -msse2 -mfma -mfpmath=sse
    cl : Command line warning D9002 : ignoring unknown option '-march=native'
    cl : Command line warning D9002 : ignoring unknown option '-msse'
    cl : Command line warning D9002 : ignoring unknown option '-msse2'
    cl : Command line warning D9002 : ignoring unknown option '-mfma'
    cl : Command line warning D9002 : ignoring unknown option '-mfpmath=sse'
    _tree.c
    c:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\lib\site-packages\numpy\core\include\numpy\npy_1_7_deprecated_api.h(12) : Warning Msg: Using deprecated NumPy API, disable it by #defining NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
    sklearn_pmml_model\tree\_tree.c(5266): warning C4244: 'function': conversion from '__pyx_t_18sklearn_pmml_model_4tree_5_tree_SIZE_t' to 'long', possible loss of data
    sklearn_pmml_model\tree\_tree.c(7433): warning C4244: '=': conversion from '__pyx_t_18sklearn_pmml_model_4tree_5_tree_SIZE_t' to '__pyx_t_18sklearn_pmml_model_4tree_6_utils_DOUBLE_t', possible loss of data
    sklearn_pmml_model\tree\_tree.c(11751): warning C4244: '=': conversion from '__pyx_t_18sklearn_pmml_model_4tree_5_tree_SIZE_t' to '__pyx_t_18sklearn_pmml_model_4tree_6_utils_DOUBLE_t', possible loss of data
    sklearn_pmml_model\tree\_tree.c(14037): warning C4244: 'function': conversion from '__pyx_t_18sklearn_pmml_model_4tree_5_tree_SIZE_t' to 'long', possible loss of data
    sklearn_pmml_model\tree\_tree.c(14101): warning C4244: 'function': conversion from '__pyx_t_18sklearn_pmml_model_4tree_5_tree_SIZE_t' to 'long', possible loss of data
    sklearn_pmml_model\tree\_tree.c(14309): warning C4244: '=': conversion from '__pyx_t_18sklearn_pmml_model_4tree_5_tree_SIZE_t' to 'long', possible loss of data
    sklearn_pmml_model\tree\_tree.c(14388): warning C4244: '=': conversion from '__pyx_t_18sklearn_pmml_model_4tree_5_tree_SIZE_t' to 'long', possible loss of data
    sklearn_pmml_model\tree\_tree.c(14965): warning C4244: 'function': conversion from '__pyx_t_18sklearn_pmml_model_4tree_5_tree_SIZE_t' to 'long', possible loss of data
    sklearn_pmml_model\tree\_tree.c(15029): warning C4244: 'function': conversion from '__pyx_t_18sklearn_pmml_model_4tree_5_tree_SIZE_t' to 'long', possible loss of data
    sklearn_pmml_model\tree\_tree.c(15331): warning C4244: '=': conversion from '__pyx_t_18sklearn_pmml_model_4tree_5_tree_SIZE_t' to 'long', possible loss of data
    sklearn_pmml_model\tree\_tree.c(15451): warning C4244: '=': conversion from '__pyx_t_18sklearn_pmml_model_4tree_5_tree_SIZE_t' to 'long', possible loss of data
    sklearn_pmml_model\tree\_tree.c(16466): warning C4133: 'function': incompatible types - from 'PyObject *' to 'PyTypeObject *'
    sklearn_pmml_model\tree\_tree.c(24679): warning C4244: 'initializing': conversion from 'double' to 'float', possible loss of data
    sklearn_pmml_model\tree\_tree.c(24685): warning C4244: 'initializing': conversion from 'double' to 'float', possible loss of data
    C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Tools\MSVC\14.16.27023\bin\HostX86\x64\link.exe /nologo /INCREMENTAL:NO /LTCG /DLL /MANIFEST:EMBED,ID=2 /MANIFESTUAC:NO /LIBPATH:c:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\libs /LIBPATH:c:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\PCbuild\amd64 "/LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Tools\MSVC\14.16.27023\lib\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.17763.0\ucrt\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.17763.0\um\x64" m.lib /EXPORT:PyInit__tree build\temp.win-amd64-3.6\Release\sklearn_pmml_model\tree\_tree.obj /OUT:build\lib.win-amd64-3.6\sklearn_pmml_model\tree\_tree.cp36-win_amd64.pyd /IMPLIB:build\temp.win-amd64-3.6\Release\sklearn_pmml_model\tree\_tree.cp36-win_amd64.lib
    LINK : fatal error LNK1181: cannot open input file 'm.lib'
    error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\BuildTools\\VC\\Tools\\MSVC\\14.16.27023\\bin\\HostX86\\x64\\link.exe' failed with exit status 1181

    ----------------------------------------
Command "c:\users\sajandeep.sidhu\appdata\local\continuum\anaconda3\python.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\SAJAND~1.SID\\AppData\\Local\\Temp\\pip-install-rww4zo40\\sklearn-pmml-model\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record C:\Users\SAJAND~1.SID\AppData\Local\Temp\pip-record-ez85ewko\install-record.txt --single-version-externally-managed --compile" failed with error code 1 in C:\Users\SAJAND~1.SID\AppData\Local\Temp\pip-install-rww4zo40\sklearn-pmml-model\

```

**Comment by iamDecode** [comment]
Hey @sajandeep thanks for letting me know. This happens because certain libraries are not available in Windows. I currently do not have the capability to test on Windows, but I pushed a fix in 1d5be9070ce3e65c2daa99c99e24990732f63104 that hopefully prevents this issue for you. Could you upgrading to 0.0.5 and try again?

**Comment by sajandeep** [comment]
Yes 0.0.5 has installed successfully for me. 
I will close this issue. 
Thank You!!
